src/main/java/edu/wisc/my/restproxy/service/RestProxyServiceImpl.java

```diff
@@ -84,6 +98,19 @@ public Object proxyRequest(final String resourceKey, final HttpServletRequest re
       .setUri(uri.toString())
       .setUsername(username);
 
+    RequestBody requestBody = null;
+    try {
+      InputStream inputStream = request.getInputStream();
+      final String contentType = request.getHeader("Content-Type");
+      if(inputStream != null && "application/json".equals(contentType)) {
```

Comment:
:+1: for constants, but since this is a very generic service, you might think about extending some love to `application/hal+json` or `text/json` as well.